### PR TITLE
[Frontend] Refactor MarketService -> ClpService

### PR DESCRIPTION
**Pull request by ryardley**
This merges the "MarketService"  with the ClpService as ClpService is all about liquidity pools. [This](https://app.asana.com/0/1198643968827245/1199548124188567) is in preparation for  [Pool: Connect List Pools API](https://app.asana.com/0/1198643968827245/1199227170208737) which is something that is holding up https://github.com/Sifchain/sifnode/pull/351

### CODE REVIEW

File: ui/core/src/actions/clp.ts [hunk 1]
```diff
@@ -5,14 +5,11 @@ import { PoolStore } from "../store/pools";
...
-}: ActionContext<
-  "SifService" | "MarketService" | "ClpService",
-  "pools" | "wallet"
->) => {
+}: ActionContext<"SifService" | "ClpService", "pools" | "wallet">) => {
   const state = api.SifService.getState();
 
   // Sync MarketService with pool store
-  api.MarketService.onPoolsUpdated((pools) => {
+  api.ClpService.onPoolsUpdated((pools) => {
```
Comment (59023g) [inline ui/core/src/actions/clp.ts:12]: I'd do either `PoolService`/`poolFinder` or `ClpService`/`ClpFinder`. For more consistency in naming

File: ui/core/src/api/ClpService/ClpService.ts [hunk 2]
```diff
@@ -4,16 +4,23 @@ import {
...
+  Pool,
 } from "../../entities";
 import { Fraction } from "../../entities/fraction/Fraction";
 
 import { SifUnSignedClient } from "../utils/SifClient";
+import { toPool } from "../utils/toPool";
+import ReconnectingWebSocket from "reconnecting-websocket";
 
 export type ClpServiceContext = {
+  loadAssets: () => Promise<Asset[]>;
```
Comment (59023g) [inline ui/core/src/api/ClpService/ClpService.ts:16]: Which assets?

**Review by 59023g — APPROVED** [review]
Looks good. Mainly `clp*` vs `pool*` naming schema

File: ui/core/src/actions/clp.ts (hunk 1 above)
Comment (ryardley) [reply ui/core/src/actions/clp.ts:12]: Yep great idea. It should be easier to get this in - get the other one in and then do a rename.

**Pull request merged**